tests/_runner.lua

```diff
@@ -89,7 +89,6 @@ runner.run_steps = function(steps, options)
                 -- Next step.
                 step = step+1
                 step_count = 0
-                wait = 5
```

Comment:
I think this needs to be reset to 20 instead.

Reply:
Oh, right. I was totally breaking things. Thanks!